ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV1.java

```diff
@@ -60,6 +64,29 @@ protected EngineStatus getInvalidBlockHashStatus() {
     return INVALID_BLOCK_HASH;
   }
 
+  @Override
+  public EngineNewPayloadRequestParameter getAndCheckEngineNewPayloadRequestParams(
+      final JsonRpcRequestContext requestContext) {
+    final EngineExecutionPayloadParameter enginePayload =
+        requestContext.getRequiredParameter(0, EngineExecutionPayloadParameter.class);
+    if (enginePayload.getWithdrawals() != null) {
+      throw new InvalidJsonRpcParameters("non-null Withdrawals pre-shanghai");
```

Comment:
This feels like some API version leakage. We don't want to have to continuously update these ancient version apis as we evolve, and add more and more parameter checks and errors.

It might be simpler to create a versioned request parameter, and add strict JSON translation, so it throws an error on unexpected properties. That way the api never needs to be modified later, but will always error out if the client tries to call it with irrelevant fields.